design-documents/frontend/technical-vision.md

```diff
@@ -0,0 +1,140 @@
+# Front-End Technical Vision
+
+The following document is a guide for all _future_ Magento front-end applications outside of core, for which we should be mindful of the following values:
+
+- Portability
+- Extensibility
+- Maintainability
+- Consistency
+- Performance
+- Accessibility
+- Internationalization
+- Productivity (Efficiency)
+- Quality (Reliability)
+
+## [React](https://reactjs.org/)

```

Comment:
I believe that better to create requirement and not provide the solution as the requirement. The almost all that you will say about "React" will be the same for Vue.Js

Reply:
Also, to be clear, the solution here and really for most of this document isn't really a set of requirements so much as it is a guide to help people sift through the plethora of technology choices available in the JS ecosystem. The first sentence of this document states that it is "a guide", so I think that should be pointed out.